src/graph.js

```diff
@@ -60,6 +61,19 @@ module.exports = (function () {
         return this;
     };
 
+    Graph.prototype.dontRender = function() {
+        this._renderNow = false;
+
+        return this;
+    };
+
+    Graph.prototype.doRender = function() {
```

Comment:
I'd rather not have these (`dontRender` and `doRender`) exposed publicly. Could you prefix their names with underscores? Unless you can think of a reason why they should be public.